src/dev/build/lib/fs.js

```diff
@@ -95,7 +95,14 @@ export async function copy(source, destination) {
   await chmodAsync(destination, stat.mode);
 }
 
-export async function deleteAll(log, patterns) {
+export async function deleteAll(patterns, log = {}) {
+  if (!log.debug || !log.verbose) {
```

Comment:
Can we instead wrap the calls to `log` in `if (log) { ... }`?

Reply:
yes we can for sure. I just chose that way to not 'pollute' each log call with an `if` wrap. However rethink about it, it's not that bad. Done!